specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -2123,6 +2124,24 @@
           "description": "Percentage completed.",
           "readOnly": true
         },
+        "value": {
+          "type": "array",
+          "items": {
```

Comment:
Might want to set readOnly:true on the items array. Might not be necessary, but probably doesn't hurt.